mantidimaging/gui/windows/spectrum_viewer/model.py

```diff
@@ -81,6 +81,8 @@ def __init__(self, presenter: SpectrumViewerWindowPresenter):
         else:
             self.tof_mode = ToFUnitMode.WAVELENGTH
 
+        self.units = UnitConversion()
```

Comment:
You could move this into `set_relevant_tof_units()` so it does not need to be an instance variable.

Reply:
The only issue with not having it as an instance variable is how to dynamically set the flight path and time delay changes, i.e. 

```
    def handle_flight_path_change(self) -> None:
        self.model.units.target_to_camera_dist = self.view.flightPathSpinBox.value()
        self.model.set_relevant_tof_units()
        self.refresh_spectrum_plot()

    def handle_time_delay_change(self) -> None:
        self.model.tof_data = self.model.get_stack_time_of_flight()
        self.model.units.set_data_offset(self.view.timeDelaySpinBox.value())
        self.model.units.data_offset = self.view.timeDelaySpinBox.value() * 1e-6
        self.model.set_relevant_tof_units()
        self.refresh_spectrum_plot()
```

we wouldn't be able to set them via `self.model.units`.

We could I suppose set them in the constructor, e.g. `__init__(self, data_to_convert, flight_path-56, time_delay=0)` in `UnitConversion` and then write `UnitConversion(self.tof_data, flight_path, time_delay).tof_seconds_to_us()` for example but for me that takes away the feeling of having the `UnitConversion` class more generalised in the future.

Reply:
My bad, I  missed those and though it was only being used from in `set_relevant_tof_units()`